src/riakc_ts.erl

```diff
@@ -122,10 +122,10 @@ put(Pid, Table, ColumnNames, Measurements)
 delete(Pid, Table, Key, Options)
   when is_pid(Pid), (is_binary(Table) orelse is_list(Table)),
        is_list(Key), is_list(Options) ->
-    Message = #tsdelreq{table   = iolist_to_binary(Table),
-                        key     = riak_pb_ts_codec:encode_cells_non_strict(Key),
-                        vclock  = proplists:get_value(vclock, Options),
-                        timeout = proplists:get_value(timeout, Options)},
+    Message = #tsdelreq{table  =iolist_to_binary(Table),
```

Comment:
These edits are mildly distracting and, well, go across the grain. Spaces around '=' are pretty much everywhere, and have won the riak world ages ago. All attempts to overthrow this style are futile.